boxwise_flask/models.py

```diff
@@ -1,28 +1,36 @@
-"""Model definitions for database"""
-from peewee import CharField, CompositeKey, DateField, DateTimeField
+"""Model definitions for database."""
+from peewee import CharField, CompositeKey, DateField, DateTimeField, IntegerField
 from playhouse.shortcuts import model_to_dict
 
-from .app import db
+from .db import db
 
 
 class Person(db.Model):
+    id = IntegerField()
+    camp_id = IntegerField()
     firstname = CharField()
     lastname = CharField()
-    camp_id = CharField()
-    id = CharField()
 
-    def __unicode__(self):
```

Comment:
@mc18g13 can u tell me why you changed this here and for the other models?

Reply:
It doesn't cast properly when you use print to see what the string representation is. It seems to use str instead of unicode.